Example/Tests/BPKRadiiTest.m

```diff
@@ -28,7 +28,9 @@ @interface BPKRadiiTest : XCTestCase
 @implementation BPKRadiiTest
 
 - (void)testRadiiHaveCorrectValues {
+    XCTAssertEqual(BPKCornerRadiusXs, 4);
     XCTAssertEqual(BPKCornerRadiusSm, 4);
+    XCTAssertEqual(BPKCornerRadiusMd, 12);
```

Comment:
I notice there is no test here for BPKCornerRadiusLg, is that intentional?

Reply:
Nope 😐